tgui/packages/tgui/interfaces/SmartVend.tsx

```diff
@@ -36,46 +45,57 @@ export const SmartVend = (props) => {
           }
         >
           {contents.length === 0 ? (
```

Comment:
this also no longer works, it would need to be `Object.keys(contents).length == 0`
yes, i really dislike that we're sending an object here, but i get why you're doing it